sensor_msgs/Image rgb_image
---
std_msgs/Float64MultiArray image_class_probability
